packages/fast-components-react-base/src/toolbar/toolbar.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import { storiesOf } from "@storybook/react";
+import React from "react";
+import Toolbar from "./";
+
+storiesOf("Toolbar", module)
+    .add("Default", () => (
+        <Toolbar>
+            <button role="button">Item 1</button>
+            <button role="button">Item 2</button>
+            <button role="button">Item 3</button>
+        </Toolbar>
+    ))
+    .add("Ignore elements without proper role set", () => (
+        <Toolbar>
+            <button role="button">Item 1</button>
+            <button>Button without a role</button>
+            <button role="button">Item 3</button>
```

Comment:
I don't think we can implement this way. The implicit role of button is already button. This implementation goes outside the current recommended standards approach of including implicit roles on semantic elements. I think we need to reconsider the approach if this is the current thinking.

Reply:
I was wondering about that.  Do we add checks for a number of semantic elements by checking their type and support those in addition to defined roles, or some other approach?

Reply:
I don't know that the current approach will work as is - with that said, I don't have an immediate resolution for how to go about it. This may be a good opportunity to gather feedback from collaborators and the community either in the Discord channels or via linking this with the spec, outlining the issue and pinging the collaborators.

Reply:
I there a role that we could assign that would exclude an element. What about `role="none"`?

https://www.scottohara.me/blog/2018/05/05/hidden-vs-none.html

Reply:
I'm not sure we should leverage the roles model to remove here, there could potentially be issues with AT by doing so. If we use role="presentation" we'll need to ensure that we treat it as such for non-AT users as well. This would be the same with something like "aria-hidden" in this instance.

Reply:
I think we can get away from the whole roles based approach by using tappable's isFocusable function or something similar.  Working on it.

Reply:
new pr on second version coming shortly